.github/workflows/autofix-ci.yml

```diff
@@ -0,0 +1,30 @@
+name: autofix.ci
+on:
+    pull_request:
+        types: [opened, synchronize, reopened, ready_for_review]
+    push:
+        branches:
+            - master
+permissions:
+    contents: write
```

Comment:
The example given in https://autofix.ci/setup only uses `read` permissions. I think you encountered an error with the `.github` directory specifically (which is fine, don't worry about that).